.github/workflows/build.yml

```diff
@@ -3,46 +3,22 @@ on: [pull_request, workflow_dispatch]
 jobs:
   cocoapods:
     name: CocoaPods (Xcode 14)
-    runs-on: macOS-12
+    runs-on: macOS-latest
```

Comment:
🧶 : keeping the explicit version here could prevent the possibility of new macOS versions breaking the CI pipeline.

Reply:
It looks like we opted for latest in the core equivalent PR: https://github.com/braintree/braintree_ios/pull/1018.

Reply:
That's a good call - down to keep the explicit versions if we think it's more stable!

Reply:
Updated here: https://github.com/braintree/braintree-ios-drop-in/pull/409/commits/a299a36aca829ffb730156f608e9c42aa400d1af